database.go

```diff
@@ -0,0 +1,1235 @@
+	if err := json.NewDecoder(resp.Body).Decode(&configuration); err != nil {
+		return nil, err
+	}
+	return configuration, nil
+}
+
+// Get a configurations.
+func (cfg *databaseConfiguration) Get(ctx context.Context, cfgID string) (*DatabaseConfiguration, error) {
+	req, err := cfg.client.NewRequest(ctx, http.MethodGet, databaseServiceName, cfg.resourcePath(cfgID), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := cfg.client.Do(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	var configuration *DatabaseConfiguration
+
+	if err := json.NewDecoder(resp.Body).Decode(&configuration); err != nil {
+		return nil, err
+	}
+
+	return configuration, nil
+}
+
+// Action with a configurations to node.
+func (cfg *databaseConfiguration) Action(ctx context.Context, nodeID string, cfgID string, iar *DatabaseAction) (*DatabaseMessageResponse, error) {
+	req, err := cfg.client.NewRequest(ctx, http.MethodPost, databaseServiceName, cfg.resourceActionPath(nodeID, cfgID), &iar)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := cfg.client.Do(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	var dmr *DatabaseMessageResponse
+
+	if err := json.NewDecoder(resp.Body).Decode(&dmr); err != nil {
+		return nil, err
+	}
+
+	return dmr, nil
+}
+
+// Update a configurations.
+func (cfg *databaseConfiguration) Update(ctx context.Context, cfgID string, cu *DatabaseConfigurationUpdate) (*DatabaseMessageResponse, error) {
+	req, err := cfg.client.NewRequest(ctx, http.MethodPut, databaseServiceName, cfg.resourcePath(cfgID), &cu)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := cfg.client.Do(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	var dmr *DatabaseMessageResponse
+
+	if err := json.NewDecoder(resp.Body).Decode(&dmr); err != nil {
+		return nil, err
+	}
+
+	return dmr, nil
+}
+
+// Delete a configurations.
+func (cfg *databaseConfiguration) Delete(ctx context.Context, cfgID string) (*DatabaseMessageResponse, error) {
+	req, err := cfg.client.NewRequest(ctx, http.MethodDelete, databaseServiceName, cfg.resourcePath(cfgID), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := cfg.client.Do(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	var dmr *DatabaseMessageResponse
+
+	if err := json.NewDecoder(resp.Body).Decode(&dmr); err != nil {
+		return nil, err
+	}
+
+	return dmr, nil
+}
+
+// Database Backup
+
+// List all backups.
+func (bk *databaseBackup) List(ctx context.Context, resource *DatabaseBackupResource, opts *DatabaseListOption) ([]*DatabaseBackup, error) {
+	resourcePath := databaseBackupsResourcePath
+	if resource != nil {
+		resourcePath = bk.resourceCreatePath(resource.ResourceType, resource.ResourceID)
+	}
+
+	req, err := bk.client.NewRequest(ctx, http.MethodGet, databaseServiceName, resourcePath, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	AddParamsListOption(req, opts)
```

Comment:
What if we use this function and want get next objects? How to do that? How to get all?